docs/_docs/publish.md

```diff
@@ -4,176 +4,51 @@ redirect_from: /docs/publish.html
 ---
 
 # Publish
-***
-Publishes a message to one or more topics.
-
-## Simple Examples   
 
+***
 
-| Command                                              |Explanation                                                              |
-|------------------------------------------------------|-------------------------------------------------------------------------|
-| ``mqtt pub -t test -m "Hello" ``                     | Publish the message ``Hello`` with topic 'test' using the default settings
-| ``mqtt pub -t test1 -t test2 -m "Hello Tests"``      | Publish the message ``Hello Tests`` with topics 'test1' and 'test2'
-| ``mqtt pub -t test -m "Hello" -h localhost -p 1884`` | Publish the message ``Hello`` with topic 'test' to a broker at localhost:1884|
-| ``mqtt pub -t test -m:file payload.txt``             | Publish the message in payload.txt with topic 'test' using the default settings
+Publishes a message to one or more topics.
 
-<!---
-See also 
-```
-mqtt pub --help
-```
--->
+***
 
-*** 
+## Simple Examples
 
-## Synopsis
-
-``` 
-mqtt pub    -t <topics> [-t <topics>]... 
```

Comment:
For some commands this overview was removed, for others not, and for some there is just the overview like `mqtt hivemq export [COMMAND]`

Reply:
My fault, I should have converted this PR to a draft. I am still working on it. 
TL; DR: Every command should now be:
- Title
- Small introduction text
- First glimpse (Command overview without any options/flags)
- Short example
- Options explained
- ...


The current concept is to remove every synopsis section as it does not add value without explanations of the flags and options. Instead, every command overview shall contain an options section with a sorted overview of all available flags and options. As a side effect, this also greatly improves documentation maintenance as the option tables can be shared across multiple pages. The HiveMQ commands page is still a bit odd, I think it is best to split it up like the shell section so subcommands are separated.